roles/configure_crontab/tasks/main.yml

```diff
@@ -1,6 +1,7 @@
 - name: Configure cron jobs
+  when: inventory_hostname == cronjob_host
```

Comment:
Hm. I'm not sure this will always reliably run on the same server - depending on which strategy you're using and where this task comes in the play, it might run on different servers on different runs, though I'm not sure either way. If this task must always run on the same specific server, then you may need to designate that server a different way.